raft/storage.go

```diff
@@ -83,6 +87,8 @@ type MemoryStorage struct {
 	snapshot  pb.Snapshot
 	// ents[i] has raft log position i+snapshot.Metadata.Index
 	ents []pb.Entry
+
+	callStats inMemStorageCallStats
```

Comment:
I do not see where `callStats` is used. Are you planning to use it in a separate PR?

Reply:
It seems also an unrelated change, so suggest to do it in a separate PR.

Reply:
Got it, it's used in the Benchmark test cases.